README.md

```diff
@@ -142,6 +142,16 @@ n.reorder_edges(e_out) # Permute final tensor as necessary
 print(tf.norm(tf.matmul(a,b) - n.get_tensor()))
 ```
 
+## Advanced examples
+Some more sophisticated examples can be found under `examples/`.
+### Trotter evolution of a wavefunction
+Demonstrates time-evolution of a wavefunction, achieved by applying a quantum circuit
+derived from a Trotter decomposition of the propagator. To run from source, use
+```
+python -m examples.wavefunctions.evolution_example
```

Comment:
nit: python3

Reply:
Not on my computer! Am happy to standardize on `python3` if that's more common.

Reply:
Nah it's fine.